packages/core/src/types/asyncapi2.ts

```diff
@@ -1099,12 +1086,32 @@ export const AsyncApi2Types: Record<string, NodeType> = {
 
   ServerBindings,
   ChannelBindings,
+  MessageBindings,
+  OperationBindings,
+};
+
+export const AsyncApi2Types: Record<string, NodeType> = {
+  ...AsyncApi2Bindings,
+  Root,
+  Tag,
+  TagList: listOf('Tag'),
+  ServerMap,
+  ExternalDocs,
+  Server,
+  ServerVariable,
+  ServerVariablesMap: mapOf('ServerVariable'),
+  SecurityRequirement,
+  SecurityRequirementList: listOf('SecurityRequirement'),
```

Comment:
should this remain consistent with other `type` files.  `NamedSecurityRequirements: listOf('SecurityRequirement')` ??

Reply:
it is consistent, I believe you miss difference between `listOf/mapOf` 😊 . It is a list of something. Please check the type below:
```
MessageTraitList: listOf('MessageTrait'),
OperationTraitList: listOf('OperationTrait'),
```

Reply:
I mean the naming of the property. 

In OAS types files, the naming is `Named*` and listOf. 

In this file you are using  `*List`

Reply:
If you find somewhere using `listOf` and constant name has pattern `Named*` please open issue or write here. I'll double-check it. 

Here is examples of using listof in OAS types:
https://github.com/Redocly/redocly-cli/blob/36b09f11d54c6f386965f5f21e2e4ca8b9be674a/packages/core/src/types/oas3.ts#L534-L550
https://github.com/Redocly/redocly-cli/blob/36b09f11d54c6f386965f5f21e2e4ca8b9be674a/packages/core/src/types/oas2.ts#L446-L479


As you could see, we use `*List` pattern for it

Reply:
ah, my apologies.  I guess I was looking at the mapOf entries. 

disregard my comments.  thanks!